player/playloop.c

```diff
@@ -662,7 +666,7 @@ static void handle_pause_on_low_cache(struct MPContext *mpctx)
 
     if (mpctx->paused_for_cache) {
         cache_buffer =
-            100 * MPCLAMP(s.ts_duration / opts->cache_pause_wait, 0, 0.99);
+            100 * MPCLAMP(s.ts_duration / cache_pause_fill, 0, 0.99);
         mp_set_timeout(mpctx, 0.2);
     }
 
```

Comment:
OK. Option name seems a bit bad.

Reply:
Suggestions welcome for better name.

Reply:
Maybe `--cache-pause-wait-max` or `--cache-pause-wait-stop`

Reply:
Another alternative would be to add a new option type like `OPT_SIZEBOX`, but for a pair of floats. So then we could use `--cache-pause-wait=0.5-1.5`, and `--cache-pause-wait=0.5` could still be parsed to mean the same thing it does today. Not sure if it makes perfect sense, and not worth doing unless it would be useful for other new options in the future.

Reply:
I'd rather prefer separate options than another case of complicated option parsing.